packages/components/src/animation/index.ts

```diff
@@ -0,0 +1,6 @@
+import AnimatePresence from './AnimatePresence';
+import LazyMotion from './LazyMotion';
+import loadFramerFeatures from './features';
+import m from './m';
+
+export default { AnimatePresence, LazyMotion, m, loadFramerFeatures };
```

Comment:
[nit] we can just export from `framer-motion` directly

```js
export { AnimatePresence, LazyMotion, m } from 'framer-motion';
```

Reply:
We need `loadFramerFeatures` too so I import them before exporting again, is that's okay? 🤗

Reply:
We can also export `loadFramerFeatures` in that way.

```js
export { loadFramerFeatures } from './features';

// packages/components/src/animation/index.ts
export { * as Animation } from './animation';
```

Reply:
Sorry I don't quite get the intention here

```javascript

// packages/components/src/animation/index.ts
export * as Animation from './animation';
```

Reply:
Also, should we update the `tsconfig.json` accordingly since we put the a 'features' into animation?

Reply:
> Sorry @arthur791004 I don't quite get the intention here

Ooops, it should be `packages/components/src/index.ts`

Reply:
> Oh, I think it's weird to put the `loadFramerFeatures` into `Animations` :joy:

Sorry @arthur791004 I got lost again 😅, 'framer-motion' is now in /components...

and import in `pattern-assembler/animate-list.tsx`

```javascript
import { Animation } from '@automattic/components';

const { AnimatePresence, LazyMotion, m, loadFramerFeatures } = Animation;
```

Reply:
Yes, I know. Let's keep the current usage.

Reply:
Thanks for the `import` tips and bearing with me on this 👍